docs/self-hosted/configuration/environment-variables.md

```diff
@@ -188,3 +188,13 @@ By default, Chatwoot will not allow users to create an account[multi-tenancy] fr
 ```bash
 ENABLE_ACCOUNT_SIGNUP=true
 ```
+
+## Enable direct upload to storage cloud
+
+By default, Chatwoot will upload the files to the application server then it will push to the cloud storage. We have introduced the direct upload functionality so that we can upload the file directly to the cloud storage, this has been built according to rails new direct upload functionality documented [here](https://edgeguides.rubyonrails.org/active_storage_overview.html#direct-uploads) . Set below environamen variable to true to use direct upload
```

Comment:
```suggestion
By default, Chatwoot will upload the files to the application server and then it will push them to the cloud storage. We have introduced the direct upload functionality so that we can upload the file directly to the cloud storage. This has been built according to rails new direct upload functionality documented [here](https://edgeguides.rubyonrails.org/active_storage_overview.html#direct-uploads). Set below environment variable to true to use the direct upload feature.
```